eoscostarica/eoscr-components
# Add Linear Progress

## Add Linear Progress 

### What does this PR do?
Add linear progress to Ricardian contract 

### How should this be tested?
- run *yarn storybook*
- go to `http://localhost:9009`
- see Ricardian contract component
